src/main/java/nl/novi/techiteasycontroller/Controller/RemoteControllerController.java

```diff
@@ -0,0 +1,14 @@
+package nl.novi.techiteasycontroller.Controller;
+
+import nl.novi.techiteasycontroller.Services.RemoteControllerService;
+import nl.novi.techiteasycontroller.Services.TelevisionService;
+
+public class RemoteControllerController {
+
+    private final RemoteControllerService remoteController;
```

Comment:
"remoteController"  is een verwarrende naam, aangezien het geen remoteController is, maar de remoteControllerService